.travis.yml

```diff
@@ -1,14 +1,14 @@
 sudo: false
 language: python
 python:
-  - "2.7"
+  - "2.7.10"
 addons:
   postgresql: "9.1"
 
 services:
   - postgresql
 install:
-  - pip install -r requirements.txt --use-mirrors
+  - pip install -r requirements.txt
```

Comment:
is there a reason not to use mirrors
(I am not sure if it's still supported or not, but I know some times it helps when installing the requirements )